R/db2.R

```diff
@@ -322,23 +322,37 @@ report_data_import <- function(con, workdir, config) {
   ## TODO: patch this back in for the saved rds I think
   hash_orderly_yml <- hash_files(file.path(workdir, "orderly.yml"), FALSE)
 
+  ## there might be a better way to this.
+  ## At the moment we are repeatedly checking if a readme has been copied
+  ## maybe add a field to the run info saying that we've copied a README?
+  if (file_exists(file.path(workdir, "README.md"), check_case = FALSE)) {
+    readme_name <- "README.md"
```

Comment:
given the case-insensitive change above, should this not find the canonical case? I think you can do this with

```r
readme_name <- file_canonical_case("README.md")
```

but you should pair that with a test that includes something like `readme.md`

Reply:
After a bit of investigation, `file_canonical_case` behaves oddly with test_that.
It looks like `.` is the test_that directory, so `file_canonical_case` will always return `NA`.